### PR TITLE
document Google authentication

### PR DESCRIPTION
**Pull request by adamblum**
@vlad-dplrn please comment and add as you see fit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -165,6 +165,25 @@ Running tests and getting test coverage:
 $ tox
 ```
 
+## Authentication with A2ML
+
+There is no explicit authentication step with A2ML. It assumes that the underlying AutoML service has been configured correctly to run on the server you are running from.  Here are abbreviated directions for that step.
```
Comment (knobotz) [inline README.md]: @adamblum this technically isn't correct. You have to authenticate with the a2ml endpoint (whether you're using the Auger Cloud A2ML endpoint or a self-hosted one). That's what the `a2ml auth login` command is for.

Comment (adamblum) [reply README.md]: oh you are right

Comment (knobotz) [reply README.md]: See my [suggested table](https://github.com/augerai/a2ml/issues/52#issuecomment-590487488) to clarify things.

Comment (adamblum) [reply README.md]: if you put that in there then you need to clarify what Auger Cloud Endpoint and Self-Hosted Endpoint mean

**Pull request merged**